py/server/deephaven/liveness_scope.py

```diff
@@ -74,5 +189,4 @@ def liveness_scope() -> LivenessScope:
         a LivenessScope
     """
     j_scope = _JLivenessScope()
-    _JLivenessScopeStack.push(j_scope)
```

Comment:
If there's existing documentation, we should be sure to note this change.

Reply:
The only docs are in this this file, as part of this PR i'll make a real how-to doc.